### DISCUSSION TITLE
[Feature request] Add clobber option to `pac modulebuilder build`

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/powerplatform-vscode#559] [Ideas]
When developing a solution sometimes an optionset/table/action/custom api can be deleted.  It would be good to have a `--clobber` option similar to `pac solution unpack` to ensure that stale files generated by previous runs of `pac modulebuilder build` are removed.

This will allow the detection of bugs faster due to existing (plugin) code that used the classes that have been removed not compiling.

A workaround is to delete Entities/OptionSets/Messages folders in the output directory before running `pac modulebuilder build`

**Comment by MattB-msft** [comment]
This feature is on our backlog to add support for.
